modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc

```diff
@@ -0,0 +1,159 @@
+:_content-type: PROCEDURE
+:description: Installing {image-puller-name-short} on OpenShift using CLI
+:keywords: administration-guide, image-puller, openshift, openshift-templates, CLI
+:navtitle: Installing {image-puller-name-short} on OpenShift using CLI
+:page-aliases: .:installing-image-puller-on-openshift-using-openshift-templates.adoc,installing-image-puller-on-openshift-using-openshift-templates.adoc
+
+[id="installing-image-puller-on-openshift-using-the-cli"]
+= Installing {image-puller-name-short} on OpenShift by using the CLI
+
+You can install the {image-puller-name} on OpenShift by using OpenShift `oc` management tool.
+
+.Prerequisites
+
+* An active `oc` session with administrative permissions to the OpenShift cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
+
+.Procedure
+
+. Gather a list of relevant container images to pull by navigating to the links:
+* `pass:c,a,q[{prod-url}]/plugin-registry/v3/external_images.txt`
+* `pass:c,a,q[{prod-url}]/devfile-registry/devfiles/external_images.txt`
+
+. Define the memory requests and limits parameters to ensure pulled containers and the platform have enough memory to run.
++
+To define the minimal value for `CACHING_MEMORY_REQUEST` or `CACHING_MEMORY_LIMIT`, consider the necessary amount of memory required to run each of the container images to pull.
++
+To define the maximal value for `CACHING_MEMORY_REQUEST` or `CACHING_MEMORY_LIMIT`, consider the total memory allocated to the DaemonSet Pods in the cluster:
++
+----
+(memory limit) * (number of images) * (number of nodes in the cluster)
+----
++
+====
+Pulling 5 images on 20 nodes, with a container memory limit of `20Mi` requires `2000Mi` of memory.
+====
+
+. Clone the {image-puller-name-short} repository and get in the directory containing the OpenShift templates:
++
+[source,bash,subs="+attributes,+quotes"]
```

Comment:
I recently learned that we're not supposed to use `bash` source language for terminal commands: https://redhat-documentation.github.io/supplementary-style-guide/#code-example-syntax-highlighting

Can you fix it, please? I noticed you using it a couple of times in the doc.